config/prow/plugins.yaml

```diff
@@ -285,6 +285,41 @@ lgtm:
   store_tree_hash: true
 
 blockades:
+- repos:
+  - kubernetes/api
+  - kubernetes/apiextensions-apiserver
+  - kubernetes/apimachinery
+  - kubernetes/apiserver
+  - kubernetes/cli-runtime
+  - kubernetes/client-go
+  - kubernetes/cloud-provider
+  - kubernetes/cluster-bootstrap
+  - kubernetes/code-generator
+  - kubernetes/component-base
+  - kubernetes/component-helpers
+  - kubernetes/controller-manager
+  - kubernetes/cri-api
+  - kubernetes/cri-client
+  - kubernetes/csi-translation-lib
+  - kubernetes/dynamic-resource-allocation
+  - kubernetes/endpointslice
+  - kubernetes/externaljwt
+  - kubernetes/kms
+  - kubernetes/kube-aggregator
+  - kubernetes/kube-controller-manager
+  - kubernetes/kube-proxy
+  - kubernetes/kube-scheduler
+  - kubernetes/kubectl
+  - kubernetes/kubelet
+  - kubernetes/metrics
+  - kubernetes/mount-utils
+  - kubernetes/pod-security-admission
+  - kubernetes/sample-apiserver
+  - kubernetes/sample-cli-plugin
+  - kubernetes/sample-controller
+  blockregexps:
+  - .*
+  explanation: "We do not accept changes directly against this repository. Please see `CONTRIBUTING.md` for information on where and how to contribute instead."
```

Comment:
Note: this is verbatim what we were posting to client-go and cri-api, just consolidated and applied to all of them.

https://github.com/kubernetes/api/blob/master/CONTRIBUTING.md